Build failure: gerbolyze

### Steps To Reproduce

Steps to reproduce the behavior:
1. build gerbolyze

### Build log

Test failures:
```
__________________________________ test_paste __________________________________

    def test_paste():
        in_gerbers = reference_path('layers-gerber')
        top_overlay = reference_path('tpl-top.svg')
        bottom_overlay = reference_path('tpl-bottom.svg')
        with tempfile.TemporaryDirectory() as intermediate_gerbers,\
                tempfile.TemporaryDirectory() as output_gerbers:
            run_command('python3', '-m', 'gerbolyze', 'paste', '--log-level', 'debug', '--no-subtract', in_gerbers, top_overlay, intermediate_gerbers)
            run_command('python3', '-m', 'gerbolyze', 'paste', '--log-level', 'debug', '--no-subtract', intermediate_gerbers, bottom_overlay, output_gerbers)
    
            stack_old = gerbonara.layers.LayerStack.open(in_gerbers)
            stack_new = gerbonara.layers.LayerStack.open(output_gerbers)
    
            for (side, use), layer_old in stack_old.graphic_layers.items():
                if use == 'outline':
                    continue
                layer_new = stack_new[side, use]
    
                bbox_old = layer_old.bounding_box(gerbonara.utils.MM)
                bbox_new = layer_new.bounding_box(gerbonara.utils.MM)
    
                e = 0.8
>               assert math.isclose(bbox_new[0][0], bbox_old[0][0]-e, abs_tol=0.1)
E               assert False
E                +  where False = <built-in function isclose>(6.142541, (6.142541 - 0.8), abs_tol=0.1)
E                +    where <built-in function isclose> = math.isclose

gerbolyze/tests/test_integration.py:91: AssertionError
```
and
```
_____________________________ test_convert_layers ______________________________

    def test_convert_layers():
        infile = reference_path('layers.svg')
        with tempfile.TemporaryDirectory() as out_dir:
            run_command('python3', '-m', 'gerbolyze', 'convert', infile, out_dir)
            stack = gerbonara.layers.LayerStack.open(out_dir)
    
            for layer, dia in {
                    'top paste':        0.100,
                    'top silk':         0.110,
                    'top mask':         0.120,
                    'top copper':       0.130,
                    'bottom copper':    0.140,
                    'bottom mask':      0.150,
                    'bottom silk':      0.160,
                    'bottom paste':     0.170,
                    'mechanical outline':    0.09}.items():
>               assert set(round(ap.diameter, 4) for ap in stack[layer].apertures()) == {dia}
E               assert set() == {0.1}
E                 
E                 Extra items in the right set:
E                 0.1
E                 Use -v to get more diff

gerbolyze/tests/test_integration.py:112: AssertionError
```

### Additional context

Upstream issue [here](https://github.com/jaseg/gerbolyze/issues/47). While upstream is working on a fix, the issue can be worked around using the shell (run `nix develop`) from the following `flake.nix`:
```
{
  inputs = {
    nixpkgs.url = "github:NixOS/nixpkgs";
    flake-utils.url = "github:numtide/flake-utils";
  };

  outputs = inputs@{ self, nixpkgs, flake-utils, ... }:
  flake-utils.lib.eachSystem [ "x86_64-linux" ] (system: let
    pkgs = nixpkgs.legacyPackages.${system};
  in rec {
    resvg-pinned = pkgs.resvg.overrideAttrs (old: rec {
      pname = "resvg";
      version = "0.41.0";
      src = pkgs.fetchFromGitHub {
        owner = "RazrFalcon";
        repo = "resvg";
        rev = "v0.41.0";
        hash = "sha256-plZiyEiBWeV2mwTsNK5Je8Axs/hcHH8aV2VpOix6QCY=";
      };
      cargoDeps = old.cargoDeps.overrideAttrs (_: {
        inherit src;
        outputHash = "sha256-2Aw+Vrn/RQUTd83gKJFMG6qEuEMl9toPCALzcPY6M4c=";
      });
    });

    gerbolyze = pkgs.gerbolyze.override{
      resvg = resvg-pinned;
    };

    devShell = pkgs.mkShell {
      buildInputs = with pkgs; [
        gerbolyze
      ];
    };
  });
}
```

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/ge/gerbolyze/package.nix
+++ b/pkgs/by-name/ge/gerbolyze/package.nix
@@ -43,6 +43,22 @@ let
       platforms = platforms.linux;
     };
   };
+
+  # FIXME: check if this downgrade is still required when bumping gerbolyze
+  # https://github.com/jaseg/gerbolyze/issues/47
+  resvg' = resvg.overrideAttrs (old: rec {
+    version = "0.41.0";
+    src = old.src.override {
+      rev = "v${version}";
+      hash = "sha256-plZiyEiBWeV2mwTsNK5Je8Axs/hcHH8aV2VpOix6QCY=";
+    };
+    cargoDeps = old.cargoDeps.overrideAttrs (lib.const {
+      name = "${old.pname}-${version}-vendor.tar.gz";
+      inherit src;
+      outputHash = "sha256-U7xzb9e9wh9XbLvlYQ0ofIjH8FuSzVcrXnrehQmZgww=";
+    });
+  });
+
 in python3Packages.buildPythonApplication rec {
   inherit version src;
   pname = "gerbolyze";
@@ -61,7 +77,7 @@ in python3Packages.buildPythonApplication rec {
     python3Packages.python-slugify
     python3Packages.lxml
     python3Packages.gerbonara
-    resvg
+    resvg'
     svg-flatten
   ];
 
@@ -79,7 +95,7 @@ in python3Packages.buildPythonApplication rec {
 
   pythonImportsCheck = [ "gerbolyze" ];
 
-  nativeCheckInputs = [ python3Packages.pytestCheckHook resvg svg-flatten ];
+  nativeCheckInputs = [ python3Packages.pytestCheckHook resvg' svg-flatten ];
 
   meta = with lib; {
     description = "Directly render SVG overlays into Gerber and Excellon files";

